app/controllers/samples_controller.rb

```diff
@@ -1136,6 +1136,8 @@ def create
 
     params[:input_files_attributes].reject! { |f| f["source"] == '' }
     params[:alignment_config_name] = AlignmentConfig::DEFAULT_NAME if params[:alignment_config_name].blank?
+    params[:status] = Sample::STATUS_CREATED
```

Comment:
I added this because it seems like a sample should always be created with a status and an owner. I'm not actually sure where this `create` method gets called though.